custom/k8s/Dockerfile

```
@@ -1 +1,14 @@
-FROM alpine/k8s:1.18.2
\ No newline at end of file
+FROM alpine:3.12
+ARG TARGETARCH
+
+LABEL maintainer="LitmusChaos"
+
+#Installing minimal packages
+RUN apk update && \
+    apk upgrade --update-cache --available 
+RUN apk --no-cache add curl &&\
+    rm -rf /var/cache/apk/*
+
+#Installing kubectl
+RUN curl -sLO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/${TARGETARCH}/kubectl" && chmod +x kubectl && mv kubectl /usr/bin/kubectl
```

```suggestion
ENV KUBECTL_VERSION="v1.19.0"
RUN curl -sLO "https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl" && chmod +x kubectl && mv kubectl /usr/bin/kubectl
```
can we use specific version in place of latest version?